lighthouse-core/audits/metrics/cumulative-long-queuing-delay.js

```diff
@@ -0,0 +1,81 @@
+/**
+ * @license Copyright 2019 Google Inc. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+ */
+'use strict';
+
+const Audit = require('../audit');
+const i18n = require('../../lib/i18n/i18n.js');
+const CumulativeLQD = require('../../computed/metrics/cumulative-long-queuing-delay.js');
+
+const UIStrings = {
+  title: 'Cumulative Long Queuing Delay',
+  description: '[Experimental metric]. Sum of Task Lengths beyond 50ms, between ' +
```

Comment:
This description seems ambiguous.  Is it the sum of tasks that are beyond 50ms in length? Or the sum of task lengths minus 50ms?

Ideas:
`Sum of task lengths that occur between FCP and TTI that delay additional task queuing by more than 50ms.`
Or make it less technical?
`Sum of tasks that delay additional task queuing.  This is an estimate of task delay based on mainthread busyness.` <- don't know if thats really true :laughing: just thinking out loud for ideas.